src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,61 @@
+.clock {
+  @extend %position;
+  @include square(500px);
```

Comment:
You can create mixin for circle too (you can reuse square mixin inside)

Reply:
yeah< thank u!!